purchase_mass_mail/view/purchase_order.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Moduon Team S.L.
+ License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+     <record id="action_purchase_order_mass_send" model="ir.actions.server">
+        <field name="name">Send email to selected purchases</field>
+        <field name="model_id" ref="purchase.model_purchase_order" />
+        <field name="binding_model_id" ref="purchase.model_purchase_order" />
+        <field name="binding_view_types">list,form</field>
+        <field name="state">code</field>
+        <field name="code">action = records.action_all_rfq_related_send()</field>
```

Comment:
Hi @shide thanks for your comment, it is not possible to use this action because is used to send only one email, for sending some emails this action doesn't work..